aptos-move/move-examples/drand/sources/drand.move

```diff
@@ -21,6 +21,8 @@
 ///     "metadata": {"beaconID": "fastnet"}
 /// }
 /// ```
+///
+/// **WARNING:** This code has NOT been audited. If using it in a production system, proceed at your own risk.
```

Comment:
we should probably just move this disclaimer to the README for the entirety of move-examples :grimacing:

Reply:
Fixed.